### PR TITLE
chore: remove accessibilityComponentType and accessibilityTraits props

### PR DESCRIPTION
**Pull request by elucaswork**
https://github.com/facebook/react-native/pull/24344

Remove deprecated `accessibilityComponentType` and `accessibilityTraits` props.

### CODE REVIEW

File: docs/view.md [hunk 1]
```diff
@@ -69,16 +69,18 @@ For `View` responder props (e.g., `onResponderMove`), the synthetic touch event
 - [`pointerEvents`](view.md#pointerevents)
 - [`removeClippedSubviews`](view.md#removeclippedsubviews)
 - [`style`](view.md#style)
-- [`testID`](view.md#testid)
-- [`accessibilityComponentType`](view.md#accessibilitycomponenttype)
+- [`testID`](view.md#testid) <<<<<<< Updated upstream
+- # [`accessibilityComponentType`](view.md#accessibilitycomponenttype)
+  > > > > > > > Stashed changes
```
Comment (charpeni) [inline docs/view.md]: This doesn't look right, @elucaswork can you look at this?

File: docs/view.md [hunk 2]
```diff
@@ -69,16 +69,18 @@ For `View` responder props (e.g., `onResponderMove`), the synthetic touch event
...
 - [`collapsable`](view.md#collapsable)
 - [`importantForAccessibility`](view.md#importantforaccessibility)
 - [`needsOffscreenAlphaCompositing`](view.md#needsoffscreenalphacompositing)
 - [`renderToHardwareTextureAndroid`](view.md#rendertohardwaretextureandroid)
 - [`accessibilityRole`](view.md#accessibilityrole)
-- [`accessibilityStates`](view.md#accessibilitystates)
-- [`accessibilityTraits`](view.md#accessibilitytraits)
+- [`accessibilityStates`](view.md#accessibilitystates) <<<<<<< Updated upstream
+- # [`accessibilityTraits`](view.md#accessibilitytraits)
+  > > > > > > > Stashed changes
```
Comment (charpeni) [inline docs/view.md]: This doesn't look right, @elucaswork can you look at this?

**Comment by cpojer** [comment]
Oops, sorry @charpeni I totally missed those. Could you resolve these?

**Comment by elucaswork** [comment]
@cpojer @charpeni oops, fix here: https://github.com/facebook/react-native-website/pull/890

**Pull request merged**